hack/generate/samples/internal/go/v3/memcached_with_webhooks.go

```diff
@@ -399,6 +459,23 @@ const reconcileFragment = `// Fetch the Memcached instance
 	}
 `
 
+const metricsFragment = `
+// MemcachedNumberOfReplicasReconcileCountTotal counts Memcached replicas number reconciliations
+var (
+	MemcachedNumberOfReplicasReconcileCountTotal = prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Name: "memcached_number_of_replicas_reconcile_count_total",
+			Help: "Total count of Memcached replicas number reconciliations",
```

Comment:
Total count of Memcached replicas number **of** reconciliations

Reply:
The `number` stands for `replicas number`. We have the `Total count` for counting the number of reconciliations.